modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java

```diff
@@ -246,7 +246,7 @@ private void _validateObjectFieldSettingFileSource(
 			String objectFieldName, String objectFieldSettingValue)
 		throws PortalException {
 
-		if (!Objects.equals(objectFieldSettingValue, "documentsAndMedia") &&
+		if (!Objects.equals(objectFieldSettingValue, "itemSelector") &&
```

Comment:
Why did you choose to change this value? Does it mean that we'll need an upgrade process to update the old values?

Reply:
@rodrigopaulino 
I thought it would be better like this because of the other story also related to the attachment field using documents and media. As this feature wasn't ready, I thought it wouldn't be a problem to change it. 
If some user saved the value "documentsAndMedia" in the database, he would be guessing that this feature would be implemented.

Should I add a upgrade process anyway? If so, maybe I will change it back to `documentsAndMedia` :stuck_out_tongue_closed_eyes:

Reply:
Got it. With this new code, what will happen if the user tries to edit a field that has this setting's value?

Reply:
Do you mean edit through the UI? Because this option was not appearing before. 
If you mean via Rest API, since the user has to send all the `objectFieldSettings` when editing, he would have to update  this setting's value.

Reply:
Exactly. Through the UI, if the user tries to edit the field, they would have to change the value, because this is a required setting?

Reply:
I don't know if I understood correctly, because through the UI it was not possible to set this value before. 
The select was always considering the `userComputer` option ([here](https://github.com/liferay-forms/liferay-portal/blob/0a6fea298063bfff7f8b7a939de859712e19d1ea/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx#L172)).

Reply:
I will change back to `documentsAndMedia`

Reply:
As discussed privately. 😄